sdk/web-pubsub/arm-webpubsub/package.json

```diff
@@ -0,0 +1,98 @@
+{
+  "name": "@azure/arm-webpubsub",
+  "sdk-type": "mgmt",
+  "author": "Microsoft Corporation",
+  "description": "A generated SDK for WebPubSubManagementClient.",
+  "version": "0.1.0-beta.1",
```

Comment:
We generally don't have version numbers that start with 0 see https://azure.github.io/azure-sdk/policies_releases.html#javascript for versioning guidelines. We would typically start with "1.0.0-beta.1"

Reply:
Thanks for sharing the link. I have a question to ask. When should we remove the beta.1 ?  when our code generator GA or when the service itself get GA. This package's api version is a preview version, I guess the service still not GA yet.

Reply:
We can cover this in the meeting I scheduled around versioning but in general we should make this when the code in the library is GA not necessarily when the service GA's. Although we don't GA a library unless the service apis we are calling are also GA.